thirdparty/patches/grpc-cython-copts.patch

```diff
@@ -20,18 +20,18 @@ diff --git bazel/cython_library.bzl bazel/cython_library.bzl
          name: Name for the rule.
          deps: C/C++ dependencies of the Cython (e.g. Numpy headers).
 +        cc_kwargs: cc_binary extra arguments such as copts, linkstatic, linkopts, features
-@@ -57,7 +59,8 @@ def pyx_library(name, deps=[], py_deps=[], srcs=[], **kwargs):
+@@ -57,7 +59,8 @@ def pyx_library(name, deps = [], py_deps = [], srcs = [], **kwargs):
 -        shared_object_name = stem + ".so"
 +        shared_object_name = stem + ".so"
          native.cc_binary(
--            name=shared_object_name,
-+            name=cc_kwargs.pop("name", shared_object_name),
--            srcs=[stem + ".cpp"],
-+            srcs=[stem + ".cpp"] + cc_kwargs.pop("srcs", []),
--            deps=deps + ["@local_config_python//:python_headers"],
-+            deps=deps + ["@local_config_python//:python_headers"] + cc_kwargs.pop("deps", []),
--            linkshared=1,
-+            linkshared=cc_kwargs.pop("linkshared", 1),
+-            name = shared_object_name,
++            name = cc_kwargs.pop("name", shared_object_name),
+-            srcs = [stem + ".cpp"],
++            srcs = [stem + ".cpp"] + cc_kwargs.pop("srcs", []),
```

Comment:
It makes sense why you added this, but do we need this change in this PR?

Reply:
We updated our gRPC dependency, so these patches were no longer applying to the correct lines, i.e. the correct code.